tests/test_facebook_automatic_fields.py

```diff
@@ -110,17 +58,8 @@ def test_run(self):
         # clear state
         menagerie.set_state(conn_id, {})
 
-        sync_job_name = runner.run_sync_mode(self, conn_id)
-
-        # verify tap and target exit codes
-        exit_status = menagerie.get_exit_status(conn_id, sync_job_name)
-        menagerie.verify_sync_exit_status(self, exit_status, sync_job_name)
-
-        # This should be validating the the PKs are written in each record
-        record_count_by_stream = runner.examine_target_output_file(self, conn_id, self.expected_sync_streams(), self.expected_pks())
-        replicated_row_count =  reduce(lambda accum,c : accum + c, record_count_by_stream.values())
-        self.assertGreater(replicated_row_count, 0, msg="failed to replicate any data: {}".format(record_count_by_stream))
-        print("total replicated row count: {}".format(replicated_row_count))
+        # run a sync
+        _ = self.run_and_verify_sync(conn_id)
 
```

Comment:
If we don't care about the record count here, then I think we should just call the function and not assign it to anything

Reply:
I did it this way so it is clear that normally there is a return value.